pkg/util/helper/policy.go

```diff
@@ -60,47 +57,18 @@ func IsDependentClusterOverridesPresent(c client.Client, policy *policyv1alpha1.
 	return true, nil
 }
 
-// GetFollowedResourceSelectorsWhenMatchServiceImport get followed derived-service and endpointSlices resource selectors
-// when policy's ResourceSelectors contains ResourceSelector, whose kind is ServiceImport.
-func GetFollowedResourceSelectorsWhenMatchServiceImport(resourceSelectors []policyv1alpha1.ResourceSelector) []policyv1alpha1.ResourceSelector {
-	var addedResourceSelectors []policyv1alpha1.ResourceSelector
-
+// ContainsServiceImport Check whether the ResourceSelectors of the policy contain ResourceSelector, and its Kind is ServiceImport.
+func ContainsServiceImport(resourceSelectors []policyv1alpha1.ResourceSelector) bool {
 	for _, resourceSelector := range resourceSelectors {
 		if resourceSelector.Kind != util.ServiceImportKind {
 			continue
 		}
-
 		if resourceSelector.Namespace == "" || resourceSelector.Name == "" {
 			continue
 		}
```

Comment:
Is it possible that this judgment is no longer necessary, and users can directly specify all ServiceImport resources?

Reply:
Updated.